pcsx2/SPU2/Mixer.cpp

```diff
@@ -373,120 +373,25 @@ __forceinline static s32 GaussianInterpolate(s32 pv4, s32 pv3, s32 pv2, s32 pv1,
 	return out;
 }
 
-/*
-   Tension: 65535 is high, 32768 is normal, 0 is low
-*/
-
-template <s32 i_tension>
-__forceinline static s32 HermiteInterpolate(
-	s32 y0, // 16.0
-	s32 y1, // 16.0
-	s32 y2, // 16.0
-	s32 y3, // 16.0
-	s32 mu  //  0.12
-)
-{
-	s32 m00 = ((y1 - y0) * i_tension) >> 16; // 16.0
-	s32 m01 = ((y2 - y1) * i_tension) >> 16; // 16.0
-	s32 m0 = m00 + m01;
-
-	s32 m10 = ((y2 - y1) * i_tension) >> 16; // 16.0
-	s32 m11 = ((y3 - y2) * i_tension) >> 16; // 16.0
-	s32 m1 = m10 + m11;
-
-	s32 val = ((2 * y1 + m0 + m1 - 2 * y2) * mu) >> 12;       // 16.0
-	val = ((val - 3 * y1 - 2 * m0 - m1 + 3 * y2) * mu) >> 12; // 16.0
-	val = ((val + m0) * mu) >> 12;                            // 16.0
-
-	return (val + (y1));
-}
-
-__forceinline static s32 CatmullRomInterpolate(
-	s32 y0, // 16.0
-	s32 y1, // 16.0
-	s32 y2, // 16.0
-	s32 y3, // 16.0
-	s32 mu  //  0.12
-)
-{
-	//q(t) = 0.5 *(    	(2 * P1) +
-	//	(-P0 + P2) * t +
-	//	(2*P0 - 5*P1 + 4*P2 - P3) * t2 +
-	//	(-P0 + 3*P1- 3*P2 + P3) * t3)
-
-	s32 a3 = (-y0 + 3 * y1 - 3 * y2 + y3);
-	s32 a2 = (2 * y0 - 5 * y1 + 4 * y2 - y3);
-	s32 a1 = (-y0 + y2);
-	s32 a0 = (2 * y1);
-
-	s32 val = ((a3)*mu) >> 12;
-	val = ((a2 + val) * mu) >> 12;
-	val = ((a1 + val) * mu) >> 12;
-
-	return (a0 + val) >> 1;
-}
-
-__forceinline static s32 CubicInterpolate(
-	s32 y0, // 16.0
-	s32 y1, // 16.0
-	s32 y2, // 16.0
-	s32 y3, // 16.0
-	s32 mu  //  0.12
-)
-{
-	const s32 a0 = y3 - y2 - y0 + y1;
-	const s32 a1 = y0 - y1 - a0;
-	const s32 a2 = y2 - y0;
-
-	s32 val = ((a0)*mu) >> 12;
-	val = ((val + a1) * mu) >> 12;
-	val = ((val + a2) * mu) >> 12;
-
-	return (val + y1);
-}
-
 // Returns a 16 bit result in Value.
 // Uses standard template-style optimization techniques to statically generate five different
 // versions of this function (one for each type of interpolation).
```

Comment:
Can delete these two comment lines as they no longer apply.

Reply:
I may be blind.